queries.sql

```diff
@@ -1,9 +1,64 @@
 
+-- First Project
+
 select * from animals where name like '%mon%';
 select name from animals where date_of_birth between '2016-01-01' and '2019-12-30';
 select name from animals where neutered = true and escape_attempt < 3;
 select date_of_birth from animals where name in ('Agumon', 'Pikachu');
 select name, escape_attempt from animals where weight_kg > 10.5;
 select * from animals where neutered = 'true';
 select * from animals where name != 'Gabumon';
-select * from animals where weight_kg between 10.3 and 17.4;
\ No newline at end of file
+select * from animals where weight_kg between 10.3 and 17.4;
+
+
+-- Second Project
+
+begin;
+update animals set species = 'unspecified';
+select * from animals;
+
+rollback;
+select * from animals;
+
+UPDATE animals 
+SET species = 'digimon'
+WHERE name like '%mon%' ;
+select * from animals;
+
+UPDATE animals 
+SET species = 'pokemon'
+WHERE species is null;
+select * from animals;
+
+commit;
+
+-- Transactions --
+
+begin;
+delete from animals;
+select * from animals;
+
+rollback;
+select * from animals;
+
+delete from animals 
+where date_of_birth > '2022-01-01';
```

Comment:
- [ ] This is a seperate transaction and as such should also start with a `begin` statement. kindly add it here as well.

[Requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md)
![image](https://user-images.githubusercontent.com/81932478/235749133-a1ee888c-152d-4a6f-b4f3-6120bb6d9055.png)